mriqc/reports/group.py

```diff
@@ -21,8 +21,6 @@
 #     https://www.nipreps.org/community/licensing/
 #
 """Encapsulates report generation functions."""
-from builtins import object  # pylint: disable=W0622
-from io import open
```

Comment:
Function `open` should be imported from `builtins`, not [`io`](https://docs.python.org/3/library/io.html#io.open):
> This is an alias for the builtin [open()](https://docs.python.org/3/library/functions.html#open) function.

But why import from `builtins` in the first place? That's implicit.

Reply:
The `future` package used to provide `builtins` for Python 2 that matched the semantics of Python 3's builtins. It's a remnant of when it was common to write code that would run on Python 2 and Python 3.